Executive.js

```diff
@@ -5,10 +5,15 @@ const BANK_COLS = 3;
 * @description Manages user interaction, updating the plan and elements on the page as required
 **/
 class Executive {
+
 	/**
 	* @post Dropdowns are populated and event listeners are set up on elements of the page the user can interact with
 	**/
-	constructor() {
+	constructor(test) {
+		// used for test suite
+		if (test) {
```

Comment:
This works for now, but we may end up needing to have all the elements on Executive uses on tests.html to be able to run write all our tests.